Debugging/NodeJs.md

```diff
@@ -0,0 +1,29 @@
+
+# NodeJs
+
+Installation instructions of the Tessel CLI can be found here: http://tessel.github.io/t2-start/
+
+This document will address common problems related to NodeJs usage.
+
+## NVM
+
+The official installation instructions assume Node is installed globally. For those of you using NVM this will not be the case. As indicated [here](https://github.com/creationix/nvm/issues/786) NVM does not install globally by design.
+
+### Linux
+
+Install `t2-cli` module as normal. To install the drivers, perform the following:
```

Comment:
On Linux, they're not technically "drivers", but [udev rules](https://www.kernel.org/pub/linux/utils/kernel/hotplug/udev/udev.html) that are installed under `/etc/` to set the permissions so that your normal user account can access the device. This can only be installed as root. You normally don't (or shouldn't) `npm install` as root, and even if you do, it drops privileges to run the build script.

On Windows, the WinUSB driver should be installed automatically from Windows Update when you plug in the device for the first time. Node / the `t2` command are not involved and you don't need to `t2 install-drivers`.